docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/mini_landing/page--cgov-mini-landing--ncids-default.html.twig

```diff
@@ -23,13 +23,10 @@
   {# Killed general-page-body-container #}
   <div class="grid-row grid-gap">
     {{ page.ncids_sidenav }}
-    <div class="cgdpl {% if page.ncids_sidenav %}desktop:grid-col-9{% else %}grid-col-12{% endif %}">
-      {# This content div is to support a lot of legacy JS #}
-      <div id="content" class="{{ page_type }}">
-        <main class="contentzone has-section-nav" id="main-content">
-          {{ page.content }}
-        </main>
-      </div>
+    <div class="{% if page.ncids_sidenav %}desktop:grid-col-9{% else %}grid-col-12{% endif %}">
+      <main class="main-content has-section-nav" id="main-content">
```

Comment:
do you need main-content for the content?? i would say it can be killed since any CSS can target `#main-content`. **I assume there is not non-legacy css targeting `.main-content`.**

Reply:
Also where is `.has-section-nav` being used?